installation-wizard/package.json

```diff
@@ -1,7 +1,8 @@
 {
   "name": "installation-wizard",
-  "version": "2.27.0",
+  "version": "2.28.0",
```

Comment:
Do we need these changes here? I am usually OK with small additional changes in PR but it looks like part of other PR.

Reply:
I upgraded the dependencies to fix 9 vulnerabilities... sorry for the noise :)